build.gradle

```diff
@@ -17,6 +18,16 @@ version = '0.0.1'
 sourceCompatibility = 1.8
 targetCompatibility = 1.8
 
+
+project.ext {
+  pactVersion = getCheckedOutGitCommitHash()
```

Comment:
why change contract version on every new merge?

Reply:
It will publish the latest contract for every commit.

Reply:
imho contract version shouldn't change if the contract doesn't change.